config.js

```diff
@@ -13,7 +13,6 @@
  */
 'use strict'; // eslint-disable-line strict
 require('./config/toggles'); // Loads the feature toggles
```

Comment:
Actually having this line here makes sure that the feature flags are loaded *before* the config is loaded.